components/hab/src/main.rs

```diff
@@ -1309,7 +1317,8 @@ async fn sub_pkg_channels(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
 async fn sub_svc_set(m: &ArgMatches<'_>) -> Result<()> {
     let remote_sup_addr = remote_sup_from_input(m)?;
     let remote_sup_addr = SrvClient::ctl_addr(remote_sup_addr.as_ref())?;
-    let service_group = ServiceGroup::from_str(m.value_of("SERVICE_GROUP").unwrap())?;
+    let service_group = required_value_of(m, "SERVICE_GROUP");
+    let service_group = ServiceGroup::from_str(service_group)?;
```

Comment:
Variable bindings

Reply:
Additionally, this would be more idiomatically written as `service_group.parse()?`, rather than `ServiceGroup::from_str(service_group)`.

From [the documentation](https://doc.rust-lang.org/std/str/trait.FromStr.html):
> FromStr's from_str method is often used implicitly, through str's parse method. See parse's documentation for examples.